uber/models.py

```diff
@@ -628,12 +628,10 @@ def change_badge(self, attendee, badge_type, badge_num=None):
                 return out_of_range
             elif not badge_num and next > c.BADGE_RANGES[badge_type][1]:
                 return 'There are no more badges available for that type'
-            elif badge_type in c.PREASSIGNED_BADGE_TYPES and c.AFTER_PRINTED_BADGE_DEADLINE:
-                return 'Custom badges have already been ordered'
 
-            if not c.SHIFT_CUSTOM_BADGES:
+            if not c.SHIFT_CUSTOM_BADGES or badge_type in c.PREASSIGNED_BADGE_TYPES and c.AFTER_PRINTED_BADGE_DEADLINE:
```

Comment:
Related fix: don't let admins assign duplicate numbers to staff badges (this was possibly when using the exploit).